src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -669,6 +679,9 @@ function GlobeControls(camera, domElement, engine) {
         spherical.setFromVector3(offset);
         state = CONTROL_STATE.NONE;
         lastRotation = [];
+        if (enableTargetHelper) {
+            this.dispatchEvent(this.changeEvent);
```

Comment:
Note: typically why I'm pretty sure the dependency between c3DEngine and Controls should not exist.